src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -268,12 +273,14 @@ export const Participants = () => {
       status,
       email,
     } = user;
+
+    const detail = details.find((detail) => detail.appletId === appletId) as RespondentDetail;
```

Comment:
Do you mean `/dashboard/respondents`? That sounds like a change best left for when we get there because that would lead to too much refactoring here. For example, when you click on a participant, the navigation to the detail screen uses the subject ID, which is applet-specific 🤷

Reply:
I think a good middle ground for now would be to fall back to the current behaviour